openedx/core/djangoapps/course_roles/models.py

```diff
@@ -0,0 +1,86 @@
+"""
+Models for course roles schema
+"""
+from django.contrib.auth import get_user_model
+from django.db import models
+
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+from organizations.models import Organization
+
+User = get_user_model()
+
+
+class CourseRolesRole(models.Model):
+    """
+    Model for a course role.
```

Comment:
These docstring comments do not really inform the reader as to how each element in the model relates to others, or to how it's intended to be used. Could you please try adding docstrings that do so?

Reply:
@julianpalmerio  Much better ... thank you. Please see additional comments I've left today